docs/source/examples/de_sscan.ipynb

```diff
@@ -0,0 +1,1498 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Python setup\n",
+    "\n",
+    "The first example will need only the bluesky RunEngine.  It will not need any\n",
+    "databroker catalog.  The EPICS IOC has a prefix of `gp:`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import bluesky\n",
+    "import bluesky.plan_stubs as bps\n",
+    "\n",
+    "RE = bluesky.RunEngine()\n",
+    "IOC = \"gp:\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Press SCAN button of a preconfigured scan\n",
+    "\n",
+    "Consider the case where the `sscan` record is already configured with all the\n",
+    "parameters necessary to perform a complete step scan.\n",
+    "\n",
+    "The next screen (left) shows `gp:scan1` is configured to step scan motor `m1`\n",
+    "from -1.2 to 1.2 in 21 points, collecting counts from scaler `gp:scaler1`\n",
+    "channels 2 & 4 (`I0` & `diode`, respectively). The next screen (right) shows\n",
+    "`gp:scaler1` is configured with a counting time of 0.2 seconds per point and\n",
+    "several detector channels.\n",
+    "\n",
+    "scaler | scan\n",
+    "--- | ---\n",
+    "![scaler1 setup](./scaler16.png) | ![scan1 setup](./sscan-scaler-v-motor.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `sscan` is preconfigured and will begin running when the *SCAN* button is\n",
+    "pressed.  When the *SCAN* button is pressed, a `1` is sent to EPICS PV\n",
+    "`gp:scan1.EXSC` and the `sscan` starts.  When the `sscan` finishes, the value of\n",
+    "the button returns to `0`.\n",
+    "\n",
+    "First, connect with the EPICS PV of the button using `ophyd.EpicsSignal`.  Once connected, show the current value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ophyd import EpicsSignal\n",
+    "\n",
+    "scan_button = EpicsSignal(f\"{IOC}scan1.EXSC\", name=\"scan_button\")\n",
+    "scan_button.wait_for_connection()\n",
+    "print(f\"{scan_button.get()}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.A. wait with a polling loop\n",
+    "\n",
+    "Write a bluesky plan that pushes the button, waits a brief moment for the scan\n",
+    "to start, then waits for the scan to end.  This simple example waits by\n",
+    "periodically polling the value of the button.\n",
+    "\n",
+    "Here, `scan_button.get(use_monitor=False)` forces ophyd to get a new value\n",
+    "from EPICS (and not to rely on the monitor value being updated promptly).\n",
```

Comment:
How can we say this clearly?  (Trying to keep this notebook from adopting a full explanation style.  I tried to make it a brief demo style.)

`SIGNAL.subscribe(callback_function)` sets up an EPICS CA monitor on SIGNAL and calls callback_function() with a dictionary of terms, including old_value and value (the ones used here).  By using `.get(use_monitor=False)`, we are demanding to get the current value from the PV directly.  This is because we are not certain the CA monitor has been received and has update the value in Python's cache.

Reply:
To be discussed live?

Reply:
discussed - we agreed to remove the `use_monitor=False` from the text and the code